extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -76,13 +71,13 @@ export function Scroller(props) {
     }
     // TODO: We should use forwardRef to dedup scrollRef and containerRef.
     const container = (scrollRef.current = containerRef.current);
-    ignoreProgrammaticScroll.current = true;
+    ignoreProgrammaticScrollRef.current = true;
     setStyle(container, 'scrollBehavior', 'auto');
     container./* OK */ scrollLeft = loop
       ? container./* OK */ offsetWidth * pivotIndex
       : container./* OK */ offsetWidth * restingIndex;
     setStyle(container, 'scrollBehavior', 'smooth');
-  }, [ignoreProgrammaticScroll, loop, restingIndex, pivotIndex, scrollRef]);
+  }, [ignoreProgrammaticScrollRef, loop, restingIndex, pivotIndex, scrollRef]);
```

Comment:
`ref` doesn't need to be in the deps now since it's defined up top.

Reply:
Good catch, thank you.